### PR TITLE
[PHP] fix php-slim4-server CI tests #4994

### LINKED ISSUE #4994: [BUG] php-slim4-server tests are failing in CI
#### Bug Report Checklist

- [ ] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [ ] What's the version of OpenAPI Generator used?
- [ ] Have you search for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

php-slim4-server tests are failing in CI
An example of it is here: https://travis-ci.org/OpenAPITools/openapi-generator/builds/636572707?utm_source=github_status&utm_medium=notification

##### Suggest a fix

It looks like we need to change the version of the slim-token-authentication library that we are using

### PR DESCRIPTION
**Pull request by spacether**
Our php-slim4-server Travis CI tests are failing on master
This PR fixes those failures by pointing to a working source for the slim-token-authentication library.

If merged, this will close this ticket:
https://github.com/OpenAPITools/openapi-generator/issues/4994

@ybelenko have I pointed to the correct branch to get this working?
It looks like you added this composer.mustache file.

- [X] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [X] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [X] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [X] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

PHP Technical committee:
@jebentier (2017/07), @dkarlovi (2017/07), @mandrean (2017/08), @jfastnacht (2017/09), @ackintosh (2017/09), @ybelenko (2018/07), @renepardon (2018/12)

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "require": {
     "php": "^7.1",
     "slim/slim": "^4.0",
-    "dyorg/slim-token-authentication": "dev-slim4",
```
Comment (spacether) [inline modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache]: This branch/release does not exist per our CI error message and:
https://packagist.org/packages/dyorg/slim-token-authentication

Comment (spacether) [reply modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache]: Per here: https://vvv.tobiassjosten.net/php/have-composer-use-development-branches/
A dev prefix lets us use a github branch

**Comment by ybelenko** [comment]
Please, try
```diff
diff --git a/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache b/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache
index 2c399a84d7..10b4ec4a10 100644
--- a/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache
+++ b/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache
@@ -2,8 +2,9 @@
   "minimum-stability": "RC",
   "repositories": [
     {
-      "type": "github",
-      "url": "https://github.com/ybelenko/slim-token-authentication"
+      "type": "vcs",
+      "url": "https://github.com/ybelenko/slim-token-authentication",
+      "no-api": true
     }
   ],
   "require": {
```

**Comment by spacether** [comment]
> Please, try
> 
> ```diff
> diff --git a/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache b/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache
> index 2c399a84d7..10b4ec4a10 100644
> --- a/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache
> +++ b/modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache
> @@ -2,8 +2,9 @@
>    "minimum-stability": "RC",
>    "repositories": [
>      {
> -      "type": "github",
> -      "url": "https://github.com/ybelenko/slim-token-authentication"
> +      "type": "vcs",
> +      "url": "https://github.com/ybelenko/slim-token-authentication",
> +      "no-api": true
>      }
>    ],
>    "require": {
> ```
> 
> It seems that Travis can't authenticate to retrieve repo, maybe option `no-api` will fix it.

How can you tell that Travis can't authenticate? What output verifies that?

**Comment by ybelenko** [comment]
```bash
11900 Failed to clone the git@github.com:ybelenko/slim-token-authentication.git
repository, try running in interactive mode so that you can enter your GitHub credentials
```
from [Travis Job #12561](https://travis-ci.org/OpenAPITools/openapi-generator/builds/636596568?utm_source=github_status&utm_medium=notification)

**Pull request merged**